packages/cli/src/oclif/commands/promote.js

```diff
@@ -113,15 +113,15 @@ ${metadataPromptHelper}`);
           .filter(({ app_change_type }) => app_change_type === 'BUGFIX')
           .map(({ issue_id }) => `#${issue_id}`);
 
-      if (appBugfixes || issueBugfixes) {
+      if (!_.isEmpty(appBugfixes) || !_.isEmpty(issueBugfixes)) {
         this.log(`Bug fixes: ${[...appBugfixes, issueBugfixes].join(', ')}`);
       }
```

Comment:
Alternatively, does something like this fix the issue? @kreddlear 

```js
if (appBugfixes || issueBugfixes) {
  this.log(`Bug fixes: ${[...(appBugfixes ?? []), ...(issueBugfixes ?? [])].join(', ')}`);
}
```

Reply:
@standielpls yes I think so! I was just playing with something like that, I just remembered the `??` operator in JS exists 😅 I asked ChangHung something similar above, but do you think that would be a cleaner solution? I'm leaning towards yes since it's a one-line change and would mean all those `isEmpty` checks don't have to apply.

Reply:
I think you're right! The `isEmpty` check is mostly important if we change from a `undefined` to `[]` so we should be able to omit those